pennylane_sf/expectations.py

```diff
@@ -42,13 +42,13 @@
 import pennylane.ops
 
 
```

Comment:
There were two options here: pass the "register" (`dev.wires`) to every expectation function and then do the extraction, or pass the `device_wires` directly. This does not work for `poly_xp`, but is still neater I think.